tests/packages/netdata/test_netdata.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import pytest
+import subprocess
+
+from lib.netutil import wrap_ip
+
+# This test installs netdata and netdata-ui packages, verifies the service status,
+# checks that the configuration is only accessible from the host, and verifies that
+# the web UI is accessible through port 19999.
+#
+# Requirements:
+# - an XCP-ng host
+
+@pytest.mark.usefixtures("host_with_netdata")
+class TestsNetdata:
+    def __get_headers(host, port, path=""):
+        url = f"http://{wrap_ip(host.hostname_or_ip)}:{port}/{path}"
+        process = subprocess.Popen(
+            ["curl", "-XGET", "-k", "-I", "-s", url],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE
+        )
+        stdout, _ = process.communicate()
+        return stdout.decode().splitlines()
+
+    # Verify the ActiveState for the netdata service
+    def test_netdata_service(self, host_with_netdata):
+        host = host_with_netdata
+        status = host.ssh(['systemctl', 'show', '--property', 'ActiveState', 'netdata'])
```

Comment:
`systemctl is-active sshd.service` would also work, but it looks already good as is.